bin/safe_commit_checks

```diff
@@ -1,26 +1,47 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
```

Comment:
noice

Reply:
just rubocop doing it's this. This repo uses this repo to do it's checks